asv_bench/benchmarks/indexing.py

```diff
@@ -191,7 +191,7 @@ def setup(self, index):
         }
         index = indexes[index]
         self.s = Series(np.random.rand(N), index=index)
-        self.indexer = [True, False, True, True, False] * 20000
+        self.indexer = list(np.random.randint(0, N, size=N))
```

Comment:
```suggestion
        self.indexer = np.random.randint(0, N, size=N)
```

(I would leave out the list here, then the focus is more on the actual take algorithm, and not on converting the list to an ndarray)

Reply:
oh, Yeah! that makes sense. I will update the PR accordingly.